include/xnnpack.h

```diff
@@ -1907,6 +1907,19 @@ enum xnn_status xnn_reshape_external_value(
   size_t num_dims,
   const size_t* dims);
 
+/// Get the external value shape.
+///
+/// @param external_id - external ID for the Value. The ID must be within the range of reversed Value IDs specified on
+///                      the Subgraph creation. The external ID can not be XNN_INVALID_VALUE_ID.
+/// @param num_dims -  A valid pointer into which the number of dimensions in the shape will be written.
+/// @param dims - pointer to an array of @a num_dims shape dimensions. This pointer can't be NULL. It must be large enough to hold
+///               at least @a num_dims elements. XNNPACK does not keep any pointers to this array after the function returns.
```

Comment:
It cannot be larger than XNN_MAX_TENSOR_DIMS

Reply:
Updated the comment.

Reply:
Please resolve merge conflict

Reply:
The PR is only changing a comment in xnnpack.h ?
Description indicates other changes... update description or re-add the other changes